contrib/vagrant/provision-full.sh

```diff
@@ -3,8 +3,9 @@ set -euo pipefail
 IFS=$'\n\t'
 USERNAME="${1:-vagrant}"
 
-yum update -y
-yum install -y docker-io git vim golang e2fsprogs tmux httpie ctags hg bind-utils which
+#yum update for fedora provider in vagrant breaks vboxfs 
```

Comment:
Sometimes I run `yum update` manually. The tip I can leave here is to use the vagrant plugin [vbguest](https://github.com/dotless-de/vagrant-vbguest) that will automatically upgrade/recompile the Guest Additions in the vm.

I agree with @danmcp that we should rather have a more up-to-date image than running this script on `vagrant up`.